libraries/botbuilder-dialogs-adaptive-testing/tests/resources/AdaptiveDialogTests/AdaptiveDialog_ParentBotInterruption.test.dialog

```diff
@@ -0,0 +1,148 @@
+{
+  "$schema": "../../../tests.schema",
```

Comment:
This seems odd, @Danieladu @chon219 @joshgummersall is there a reason we don't have a central version of this `test.schema` hosted somewhere? (Like https://schemas.botframework.com/schemas/component/v1.0/component.schema)

Not blocking on this comment, just wondering if this can be tightened up. Especially since the JS SDK's AdaptiveDialog/Declarative parity is closer than it's ever been with the .NET SDK.